src/main/java/uk/sky/cqlmigrate/CqlMigratorImpl.java

```diff
@@ -105,20 +116,25 @@ public void migrate(Session session, String keyspace, Collection<Path> directori
     /**
      * {@inheritDoc}
      */
-    public void clean(String[] hosts, int port, String username, String password, String keyspace) {
-        try (Cluster cluster = CassandraClusterFactory.createCluster(hosts, port, username, password);
-             Session session = cluster.connect()) {
-            this.clean(session, keyspace);
+    public void clean(String[] hosts, String localDC, int port, String username, String password, String keyspace) {
+
+        List<InetSocketAddress> cassandraHosts = Stream.of(hosts).map(host -> new InetSocketAddress(host, port)).collect(Collectors.toList());
+
+        try (CqlSession cqlSession = CqlSession.builder()
```

Comment:
(mentioned earlier) no longer using the factory to create the session; deliberate?